spec/models/aroc_week_2_spec.rb

```diff
@@ -15,11 +15,12 @@
     expected_result = [@order_3, @order_15, @order_9, @order_12]
 
     # ----------------------- Using Ruby -------------------------
-    orders_of_user_3 = Order.all.select { |order| order.user_id == @user_3.id }
+    # orders_of_user_3 = Order.all.select { |order| order.user_id == @user_3.id }
     # ------------------------------------------------------------
 
     # ------------------ Using ActiveRecord ----------------------
     # Solution goes here
+    orders_of_user_3 = Order.where(user_id: @user_3.id)
```

Comment:
This will work, however it looks like in the user model that a User has_many orders. So instead you could do `@user_3.orders`. They will do the same thing in sql, but the association is the preferred syntax.